compose/environment-variables/envvars.md

```diff
@@ -119,6 +121,27 @@ When enabled, Compose doesn't try to detect orphaned containers for the project.
 
 Specifies the maximum level of parallelism for concurrent engine calls.
 
+### COMPOSE\_ANSI
+
+Specifies when to print ANSI control characters. 
+
+* Supported values:
+  * `auto`, Compose detect if TTY mode can be use otherwise use plain text mode,
+  * `never`, use plain text mode.
+  * `always` or `0`, use TTY mode.
+* Defaults to: `auto`.
+
+### COMPOSE\_STATUS\_STDOUT
+
+When enabled, Compose will switch the output stream, with status and progress messages, to `stdout` instead of `stderr`.
+The default value is false to use Posix `stderr` in order to easily separate Compose output stream 
```

Comment:
I'm not 100% sure what this sentence means. Is there a clearer way to write this?

Reply:
Perhaps something like:

> By default, Compose writes all informational and progress messages to `stderr` and only writes final command output to `stdout` so that it can be used in scripts or chained with other commands.

Longer explanation/example:
```shell
❯ docker compose ps --format=json | jq '.[].ID'
"03045c92f2bfb06fd1d2a82462741420777cd62f35afd078c1410e87a50f7a1c"
```
Pipes pass `stdout` so this makes sure that the output being piped is as expected, i.e. valid JSON in this case.

If you ran the command from a terminal, you might see something like:
```shell
❯ docker compose ps --format=json
WARN: this is very important information you need to know
[{"ID": "03045c92f2bfb06fd1d2a82462741420777cd62f35afd078c1410e87a50f7a1c", ...}]
```
If the informational message was writing to `stdout`, then when piped to `jq`, it would fail.

(I don't think we need to go into this level of detail in the docs, just providing some deeper context to help formulate better concise explanation 😅)

Reply:
@aevesdocker feel free to update the PR and merge it when you're happy

Reply:
And @glours has it with the winning sentence!
(Thank you @milas for the added eplaino)